dev/com.ibm.ws.microprofile.faulttolerance.spi/src/com/ibm/ws/microprofile/faulttolerance/spi/ExecutorBuilder.java

```diff
@@ -92,4 +92,12 @@
      * @return the new executor
      */
     public <W> Executor<W> buildAsync(Class<?> asyncResultWrapperType);
+
+    /**
+     * Set a request context bean so that the request context can be activated and deactivated during execution.
+     *
+     * @param asyncRequestContext an injected instance of the application scoped AsyncRequestContextControllerImpl class (the bean).
```

Comment:
When you're passing the interface as a parameter, you don't want to be talking about the interface in the documentation for it. It should work for _any_ valid implementation of the interface.